include/aie/Dialect/AIE/IR/AIEOps.td

```diff
@@ -758,6 +758,7 @@ def AIE_DMABDOp: AIE_Op<"dma_bd", [
         2. the transfer length;
         3. the sizes and strides for n-d tensor addressing (described below);
         4. the "bd_id" with which to associate the buffer descriptor (most often left empty).
+        5. the number of zeros to pad before and after every dimension of an n-d tensor (described below);
 
     `offset`, `len`, `size`s and `stride`s are all denominated in element width; e.g., transferring the whole of
```

Comment:
Clarify that this is in units of the element width, like everything else?

Reply:
Description updated.